src/common/transformations/include/transformations/common_optimizations/mark_exp_reduceop_to_keep_in_mixed_precision.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/graph_rewrite.hpp"
+#include "openvino/pass/pass.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+namespace pass {
+
+class TRANSFORMATIONS_API MarkExpReduceOpToKeepInMixedPrecision;
+
+}  // namespace pass
+}  // namespace ov
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief: MarkExpReduceOpToKeepInMixedPrecision  marks path that goes
+ * into ReduceSum and ReduceMean. Values that go from Exp to ReduceSum/ReduceMean are precision
+ * sensitive and such nodes should be kept in f32 precision for mixed inference.
+ */
+class ov::pass::MarkExpReduceOpToKeepInMixedPrecision : public ov::pass::BackwardGraphRewrite {
```

Comment:
Please also consider renaming to `MarkReduceOfExponent`

Reply:
renamed to `MarkExpReduceOp`